code/modules/research/mechanic/flatpack.dm

```diff
@@ -147,6 +147,13 @@
 		if((stacked.len + stacking.stacked.len + 2) >= MAX_FLATPACK_STACKS) //how many flatpacks we can in a stack (including the bases)
 			to_chat(user, "You can't stack flatpacks that high.")
 			return
+		if(user.incapacitated() || user.lying)
```

Comment:
Also @Probe1 when he says make a proc, he means make a proc of this:
```
		if(user.incapacitated() || user.lying)
			return //Validate mob status
		if(!isturf(user.loc) || !Adjacent(user))
			return //Validate location, and distance to location and object
		if(!ishuman(user) && !isrobot(user))
			return //Validate mob type
```

Good rule of thumb is that if you have to copypaste it once, you should make it a proc.

I'd suggest an atom level can_MouseDrop() that just returns by default